ui/components/multichain/import-tokens-modal/import-tokens-modal.test.js

```diff
@@ -189,6 +189,13 @@ describe('ImportTokensModal', () => {
       );
       expect(getByText('Next')).not.toBeDisabled();
 
+      // wait for the symbol input to be in the document
+      await waitFor(() =>
```

Comment:
The use of `useEffect` above now causes `import-tokens-modal-custom-symbol` to be available only when the `tokenList` has loaded